recipes/openblas/all/CMakeLists.txt

```diff
@@ -8,4 +8,4 @@ else()
 endif()
 conan_basic_setup()
 
-add_subdirectory("source_subfolder")
+add_subdirectory("src")
```

Comment:
Probably better to leave a standard name for sub-directories but enable `short_paths = True` to mitigate a path length problem.

Reply:
Thank you for your comment!
I am going to try it.

Reply:
fixed